cfme/cloud/instance/__init__.py

```diff
@@ -60,6 +60,18 @@ class images(Accordion):  # noqa
         tree = ManageIQTree()
 
 
+class InstanceCompareAccordion(View):
+    """
+    The accordion on the instance comparison page
+    """
+    @View.nested
+    class comparison_sections(Accordion):       # noqa
+        ACCORDION_NAME = 'Comparison Sections'
+        tree = CheckableBootstrapTreeview()
+
+    apply_button = Button('Apply')
```

Comment:
Remove `_button` suffix from widgets, we'd like to not have that anywhere unless needed to distinguish between two similar widgets in a view.